### ISSUE TITLE
[4.0] Notice when displaying Articles Options

### ISSUE THREAD

**Issue by infograf768** [joomla/joomla-cms#19883]
### Steps to reproduce the issue

Go to administrator/index.php?option=com_config&view=component&component=com_content

Page displays apparently fine but we get a Notice twice
```
[10-Mar-2018 18:29:57 Europe/Berlin] PHP Notice:  Undefined offset: 0 in /Applications/MAMP/htdocs/joomla40/layouts/joomla/form/field/radio.php on line 71
[10-Mar-2018 18:29:57 Europe/Berlin] PHP Notice:  Trying to get property of non-object in /Applications/MAMP/htdocs/joomla40/layouts/joomla/form/field/radio.php on line 71
[10-Mar-2018 18:29:57 Europe/Berlin] PHP Notice:  Undefined offset: 1 in /Applications/MAMP/htdocs/joomla40/layouts/joomla/form/field/radio.php on line 72
[10-Mar-2018 18:29:57 Europe/Berlin] PHP Notice:  Trying to get property of non-object in /Applications/MAMP/htdocs/joomla40/layouts/joomla/form/field/radio.php on line 72
[10-Mar-2018 18:29:57 Europe/Berlin] PHP Notice:  Undefined offset: 0 in /Applications/MAMP/htdocs/joomla40/layouts/joomla/form/field/radio.php on line 71
[10-Mar-2018 18:29:57 Europe/Berlin] PHP Notice:  Trying to get property of non-object in /Applications/MAMP/htdocs/joomla40/layouts/joomla/form/field/radio.php on line 71
[10-Mar-2018 18:29:57 Europe/Berlin] PHP Notice:  Undefined offset: 1 in /Applications/MAMP/htdocs/joomla40/layouts/joomla/form/field/radio.php on line 72
[10-Mar-2018 18:29:57 Europe/Berlin] PHP Notice:  Trying to get property of non-object in /Applications/MAMP/htdocs/joomla40/layouts/joomla/form/field/radio.php on line 72
```

**Comment by infograf768** [comment]
If I first test for $options, I do not get the Notice.
I.e.
```
		<?php if (!empty($options)) : ?>
			off-text="<?php echo $options[0]->text; ?>"
			on-text="<?php echo $options[1]->text; ?>"
			<?php echo $disabled ? 'disabled' : '';?>>
		<?php endif; ?>
```

**Comment by Quy** [comment]
The affected settings:
```
		<joomla-field-switcher
		id="jform_list_show_votes"
				off-text="<br />
<b>Notice</b>:  Undefined offset: 0 in <b>C:\xampp\htdocs\joomla4\layouts\joomla\form\field\radio.php</b> on line <b>71</b><br />
<br />
<b>Notice</b>:  Trying to get property 'text' of non-object in <b>C:\xampp\htdocs\joomla4\layouts\joomla\form\field\radio.php</b> on line <b>71</b><br />
"
		on-text="<br />
<b>Notice</b>:  Undefined offset: 1 in <b>C:\xampp\htdocs\joomla4\layouts\joomla\form\field\radio.php</b> on line <b>72</b><br />
<br />
<b>Notice</b>:  Trying to get property 'text' of non-object in <b>C:\xampp\htdocs\joomla4\layouts\joomla\form\field\radio.php</b> on line <b>72</b><br />
"
		>
```
```
		<joomla-field-switcher
		id="jform_list_show_ratings"
				off-text="<br />
<b>Notice</b>:  Undefined offset: 0 in <b>C:\xampp\htdocs\joomla4\layouts\joomla\form\field\radio.php</b> on line <b>71</b><br />
<br />
<b>Notice</b>:  Trying to get property 'text' of non-object in <b>C:\xampp\htdocs\joomla4\layouts\joomla\form\field\radio.php</b> on line <b>71</b><br />
"
		on-text="<br />
<b>Notice</b>:  Undefined offset: 1 in <b>C:\xampp\htdocs\joomla4\layouts\joomla\form\field\radio.php</b> on line <b>72</b><br />
<br />
<b>Notice</b>:  Trying to get property 'text' of non-object in <b>C:\xampp\htdocs\joomla4\layouts\joomla\form\field\radio.php</b> on line <b>72</b><br />
"
		>
```

**Comment by C-Lodder** [comment]
Please test with: https://github.com/joomla/joomla-cms/pull/19296

**Comment by Quy** [comment]
I no longer get the notices after the merged of #19296.

**Comment by infograf768** [comment]
Indeed. Let's close it
